build.fsx

```diff
@@ -79,11 +209,68 @@ let packages =
     }
   ]
 
-Target.create "BuildVersion" (fun _ -> 
+type HaveUpdatedBuildVersion = | HaveUpdatedBuildVersion
```

Comment:
By the way, nothing ever seems to call this, but I've left it in.

Reply:
That is fine. Leftover from source linking or some such I believe. Which stopped working at some point, but hopefully is easier to bring back with the nuget integration now.